lib/modules/manager/gradle/parser.ts

```diff
@@ -230,6 +230,7 @@ function processPlugin({
       const fileReplacePosition = varData.fileReplacePosition;
       dep.currentValue = currentValue;
       dep.managerData = { fileReplacePosition, packageFile };
+      dep.groupName = varData.key;
```

Comment:
Please, handle similar case for `pluginVersion.type === TokenType.StringInterpolation` below